src/libespeak-ng/synthdata.c

```diff
@@ -329,7 +329,7 @@ unsigned char *GetEnvelope(int index)
 	return (unsigned char *)&phondata_ptr[index];
 }
 
-static void SetUpPhonemeTable(int number, bool recursing)
```

Comment:
This is probably wrong, because previously it had that parameter used:
```
static void SetUpPhonemeTable(int number, bool recursing)
{
	int ix;
	int includes;
	int ph_code;
	PHONEME_TAB *phtab;

	if (recursing == false)
		memset(phoneme_tab_flags, 0, sizeof(phoneme_tab_flags));

	if ((includes = phoneme_tab_list[number].includes) > 0) {
		// recursively include base phoneme tables
		SetUpPhonemeTable(includes-1, true); // Here!
	}
...
```

Reply:
Commits 713e8a9e2c and 65847bd251004 could be squashed together.

In 65847bd251004:
`phoneme_tab_flags`  is a global but only used in SetUpPhonemeTable(), it's set twice but never checked for anything.. It's unnecessary and should be removed.

 713e8a9e2c:
`recursing` is only used to set `phoneme_tab_flags. Since those if-clauses are unnecessary, so is providing the parameter.

I think SetUpPhonemeTable() should have a for loop instead of it being  recursive function, but that's a problem for another commit.